setup.py

```diff
@@ -40,10 +40,12 @@
         "sentry": "sentry-sdk>=0.14.4",
         "spooky": "spooky>=2.0.0",
         "test": [
+            "nose>=1.3.7",
             "sentry-sdk>=0.14.4",
             "PyHamcrest",
             "coverage",
-            "parameterized"
+            "parameterized",
+            "microcosm-metrics>=2.5.1"
```

Comment:
See test setup in `microcosm_flask/tests/test_metrics.py`

If we do not import `microcosm-metrics` - metrics tests are skipped.